### ISSUE TITLE
"SKIP_FAIL2BAN" Directive Should be Handled Internally in the Web UI

### ISSUE THREAD

**Issue by PhoenixPeca** [mailcow/mailcow-dockerized#907]
Hi @andryyy,

First, I have to say that having the fail2ban feature in mailcow is awesome! However, I believe that the "SKIP_FAIL2BAN" directive in mailcow config file is an unnecessary directive.

As far as I can tell, we can completely disable fail2ban in _Fail2ban parameters_ in the mailcow UI by just adding the following lines (whitelist all IP) in the **Whitelisted networks/hosts** textarea:
````
::/0
0.0.0.0/0
````

Furthermore, (If my assumptions are correct) setting `SKIP_FAIL2BAN=y` in the config file will just render the **Fail2ban parameters** panel in the mailcow UI pointless and would just leave the admins confused.

Besides, having both the ability to disable and/or enable/utilize the fail2ban feature in the mailcow UI and in the mailcow config file would be a redundant feature.

________________________

So I propose you to:
1. Dishonor the "SKIP_FAIL2BAN" directive in the config file; and
2. maybe add a toggle to the panel that automatically adds the whitelist all IP directive automatically to disable fail2ban; or publish a documentation that tackles about disabling fail2ban completely by whitelisting all possible IP addresses.

_______________________

For more info, please check [this](https://en.wikipedia.org/wiki/0.0.0.0#As_a_default_route)

I'm just sayin' though. This isn't really a major issue. 😉

**Comment by andryyy** [comment]
Yes, that's a great idea. At the same time we can think about renaming it and adding a source ip feature.
